### PR TITLE
Revision of Applications section transliterated output

### LINKED ISSUE #65: Transliterated application list
https://github.com/ubuntu-mate/mate-menu/blob/28c9fbb47bf7a3b0a28f8d119c84842a74ddf499/mate_menu/easybuttons.py#L401

unidecode turns Russian (or any non-Latin language) into unreadable mishmash. What is the general purpose of strip_accents? Can we completely remove it from the module?

Compare: on the first image how it looks at the moment, on the second image what I want to get.

Снимок экрана в 2019-10-31 17-27-04
Снимок экрана в 2019-10-31 17-27-25

### PR DESCRIPTION
**Pull request by MikePooh**
Fixes #65

### CODE REVIEW

**Comment by flexiondotorg** [comment]
This commit is relevant to reviewing this pull-request @vkareh :smiley:

File: mate_menu/easybuttons.py [hunk 1]
```diff
@@ -408,7 +408,7 @@ def filterText( self, text ):
         return True
 
     def strip_accents(self, value):
-        return unidecode.unidecode(value)
+        return value
```
Comment (mymedia2) [inline mate_menu/easybuttons.py]: Now the strip_accents method is unneeded and you can remove it and all its calls in this file.

Comment (mymedia2) [reply mate_menu/easybuttons.py]: And you can also remove unidecode dependency as it is no longer used.

**Comment by flexiondotorg** [comment]
Thanks for contributing @MikePooh :smiley:

**Pull request merged**